ui/src/components/layouts/HalfMapLayout/NewMap.tsx

```
@@ -0,0 +1,416 @@
+//   const map = useMap();
+//   useEffect(() => {
+//     map.on("click", onClick);
+//     return () => {
+//       map.off("click", onClick);
+//     };
+//   }, [map, onClick]);
+
+//   return null;
+// }
+
+export default function Map() {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)